Source/OxyPlot/Svg/SvgRenderContext.cs

```diff
@@ -56,15 +55,10 @@ public SvgRenderContext(Stream s, double width, double height, bool isDocument,
         }
 
         /// <summary>
-        /// Gets or sets the text measurer.
+        /// Gets or sets the <see cref="TextArranger"/> used by this instance.
         /// </summary>
-        /// <value>The text measurer.</value>
-        public IRenderContext TextMeasurer { get; set; }
-
-        /// <summary>
-        /// Gets or sets a value indicating whether to use a workaround for vertical text alignment to support renderers with limited support for the dominate-baseline attribute.
-        /// </summary>
-        public bool UseVerticalTextAlignmentWorkaround { get; set; }
+        /// <value>The text arranger.</value>
+        public TextArranger TextArranger { get; set; }
```

Comment:
Could we add this to `RenderContextBase`?

Reply:
I'm on the fence about this. I'll change it, then we can see how we feel after that.

Reply:
Without pulling the `ITextMeasurer` chunk of each render context into a new class or abstractly implementing `ITextMeasurer` from `RenderContextBase`, this would be a tad unpleasant. Such a change would probably make sense, but I'd sooner keep the current design, and leave that for another PR.